src/main/java/underdogs/devbie/notice/controller/NoticeController.java

```diff
@@ -0,0 +1,68 @@
+package underdogs.devbie.notice.controller;
+
+import java.net.URI;
+
+import javax.validation.Valid;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.RequiredArgsConstructor;
+import underdogs.devbie.notice.dto.NoticeCreateRequest;
+import underdogs.devbie.notice.dto.NoticeDetailResponse;
+import underdogs.devbie.notice.dto.NoticeResponses;
+import underdogs.devbie.notice.dto.NoticeUpdateRequest;
+import underdogs.devbie.notice.service.NoticeService;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/notices")
+public class NoticeController {
+
+    private final NoticeService noticeService;
+
+    @PostMapping
+    public ResponseEntity<Void> save(@Valid @RequestBody NoticeCreateRequest request) {
+        Long noticeId = noticeService.save(request);
+        return ResponseEntity
+            .created(URI.create(String.format("/api/notices/%d", noticeId)))
+            .build();
+    }
+
+    @PatchMapping("/{id}")
+    public ResponseEntity<Void> update(@PathVariable Long id,
+        @Valid @RequestBody NoticeUpdateRequest request) {
+        noticeService.update(id, request);
+        return ResponseEntity
+            .noContent()
+            .build();
+    }
```

Comment:
```suggestion
    @PatchMapping("/{id}")
    public ResponseEntity<Void> update(
        @PathVariable Long id,
        @Valid @RequestBody NoticeUpdateRequest request
    ) {
        noticeService.update(id, request);
        return ResponseEntity
            .noContent()
            .build();
    }
```
전 개인적으로 이런 방식의 띄어쓰기가 눈에 잘들어오는데 어떻게 생각하시나요
이 부분도 팀적으로 컨벤션을 정하고 싶어요

Reply:
(회의 결과)컨트롤러에서 어노테이션 쓰는 인자가 두 개 이상인 경우 띄어쓰는 방식으로 통일🙆‍♂️